pkg/util/aad/aad.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/Azure/go-autorest/autorest/adal"
 	"github.com/Azure/go-autorest/autorest/azure/auth"
-	"github.com/form3tech-oss/jwt-go"
+	"github.com/golang-jwt/jwt/v4"
```

Comment:
why bother to change when this file is going away?

Reply:
it was part of a rebase yesterday.  this will go away with the new rebase.